strawberryfields/backends/tfbackend/ops.py

```diff
@@ -42,7 +42,7 @@
     squeeze_parity,
 )
 
-def_type = tf.complex64
+def_type = tf.complex64  # NOTE: what if a user wants higher accuracy?
```

Comment:
Solution to this could be passing the precision as a keyword argument when choosing the `tf` backend? (Out of scope for now I'd guess)

Reply:
Yep, I would say this counts as a 'new feature' and thus out of scope for now!